frontend/src/metabase/visualizations/echarts/cartesian/model/dataset.unit.spec.ts

```diff
@@ -577,18 +580,5 @@ describe("dataset transform functions", () => {
       expect(result[1][X_AXIS_DATA_KEY]).toBe(5);
       expect(result[2][X_AXIS_DATA_KEY]).toBe(1000);
     });
-
-    it("handles empty datasets without errors", () => {
```

Comment:
What's the context for this test being deleted?

Reply:
It looks like this test here is the result of some rebase in the past because it was in the `describe("getTransformedDataset", () => {})` group while testing `applyVisualizationSettingsDataTransformations` which is a newer name. And there is already a "should work on empty datasets" test of `applyVisualizationSettingsDataTransformations`